policies/technical/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: OpenSSL Technical Policies
+subtitle: |
+    First issued 30th September 2020
+    <br />
+    Moved to GitHub Nov 2021
+---
+These are the technical policies and procedures established by the OTC
+in accordance with the [project bylaws]
```

Comment:
Again a colon at the end to introduce the list properly.

Reply:
Or a period...